pom.xml

```diff
@@ -27,151 +32,166 @@
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-xml</artifactId>
 		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-stream-test-support</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-stream-test-support-internal</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-test</artifactId>
+		</dependency>
+
 	</dependencies>
 
-  <properties>
-    <scs-app-maven-plugin.version>1.0.0.BUILD-SNAPSHOT</scs-app-maven-plugin.version>
-    <java.version>1.7</java.version>
-    <spring-cloud-stream.version>1.0.0.RC3</spring-cloud-stream.version>
-    <spring-integration.version>4.2.5.RELEASE</spring-integration.version>
-    <spring-integration-java-dsl.version>1.1.2.RELEASE</spring-integration-java-dsl.version>
-  </properties>
+	<properties>
+		<scs-app-maven-plugin.version>1.0.0.BUILD-SNAPSHOT</scs-app-maven-plugin.version>
+		<java.version>1.7</java.version>
+		<spring-cloud-stream.version>1.0.0.RC3</spring-cloud-stream.version>
+		<spring-integration.version>4.2.5.RELEASE</spring-integration.version>
+		<spring-integration-java-dsl.version>1.1.2.RELEASE</spring-integration-java-dsl.version>
```

Comment:
`spring-cloud-stream-app-dependencies` is the main BOM that will be used by the binder specific apps and we need those properties there. We probably can clean things up in the root pom. The only thing thats needed there is the maven plugin version.